Telemetry/TelemetryUtils.swift

```diff
@@ -57,7 +60,33 @@ class TelemetryUtils {
         return string
     }
 
-    static func daysBetween(start: Date, end: Date) -> Int {
-        return Calendar.current.dateComponents([.day], from: start, to: end).day ?? 0
+    static func daysOld(date: Date) -> Int {
+        let end = TelemetryUtils.dateFromTimestamp(TelemetryUtils.timestamp())
+        return Calendar.current.dateComponents([.day], from: date, to: end).day ?? 0
     }
 }
+
+// Allows for adjusting the time when testing
+extension TelemetryUtils {
+    static var mockableOffset: TimeInterval? {
```

Comment:
This is the important change in this file, and the rest is updating to use TelemetryUtils timestamp functions.